Data_storage/client_minio.py

```diff
@@ -0,0 +1,64 @@
+from minio import Minio
+from pathlib import Path
+import unittest
+
+ACCESS_KEY = "minioadmin"
+SECRET_KEY = "minioadmin"
+ENDPOINT = "0.0.0.0:9000"
+
+class Minio_client():
+    def __init__(self):
+        try:
+            self.minioClient = Minio(ENDPOINT, access_key = ACCESS_KEY, secret_key = SECRET_KEY, secure = False)
+        except:
```

Comment:
please add exception type, by default it uses except Exception, which is very broad type of Exceptions and not recommended to use